package.json

```diff
@@ -34,6 +34,7 @@
     "chromecast-player": "^0.2.3",
     "debounced-seeker": "^1.0.0",
     "debug": "^2.1.0",
+    "diveSync": "^0.3.0",
```

Comment:
In fact, I've just realised that there's no need to rewrite `isDir()`, because the methods used there are part of base `fs`, not `fs-extended`.